charts/stable/common/templates/_daemonset.tpl

```diff
@@ -3,32 +3,33 @@ This template serves as the blueprint for the DaemonSet objects that are created
 within the common library.
 */}}
 {{- define "common.daemonset" -}}
-{{- print ("---\n") | nindent 0 -}}
-apiVersion: {{ include "common.capabilities.daemonset.apiVersion" . }}
+  {{- print ("---\n") | nindent 0 -}}
+apiVersion: apps/v1
```

Comment:
I know why you did this (because proper indenting is 👍), but I'm not sure how much more readable it becomes when the text that gets rendered by Helm can't be indented causing this mixed indenting 😞

Reply:
Wait what do you mean the text cant be indented? The whitespace chomp will ensure it is _not_ actually indented. Unless you mean why is it not just
```
{{- print "---" }}
```
in which case, it should be... that was there before and i missed cleaning it :(

Reply:
No, I mostly meant the "misindent" between the `print` line and the `apiVersion` line. But I don't think that can really be fixed

Reply:
Switched to plain ```